src/libraries/Microsoft.PowerFx.Connectors/Tabular/Capabilities/ColumnCapabilities.cs

```diff
@@ -12,19 +12,31 @@
 
 namespace Microsoft.PowerFx.Connectors
 {
-    internal sealed class ColumnCapabilities : ColumnCapabilitiesBase, IColumnsCapabilities
+    public sealed class ColumnCapabilities : ColumnCapabilitiesBase, IColumnsCapabilities
     {
         [JsonInclude]
         [JsonPropertyName(CapabilityConstants.ColumnProperties)]
         [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingNull)]
         public Dictionary<string, ColumnCapabilitiesBase> Properties => _childColumnsCapabilities.Any() ? _childColumnsCapabilities : null;
 
-        private readonly Dictionary<string, ColumnCapabilitiesBase> _childColumnsCapabilities;
+        private Dictionary<string, ColumnCapabilitiesBase> _childColumnsCapabilities;
 
         [JsonInclude]
         [JsonPropertyName(Constants.XMsCapabilities)]
         [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingNull)]
-        public readonly ColumnCapabilitiesDefinition Capabilities;
+        public ColumnCapabilitiesDefinition Capabilities;
+
+        public static readonly IEnumerable<string> DefaultFilterFunctionSupport = new string[] { "eq", "ne", "gt", "ge", "lt", "le", "and", "or", "cdsin", "contains", "startswith", "endswith", "not", "null", "sum", "average", "min", "max", "count", "countdistinct", "top", "astype", "arraylookup" };
+
+        public static ColumnCapabilities DefaultCdsColumnCapabilities => new ColumnCapabilities()
```

Comment:
>DefaultCdsColumnCapabilities [](http://example.com/codeflow?start=41&length=28)

Do these match with capabilities PA understands as default? Just trying to make sure that code remains consistent here.